### PR TITLE
Initial layout functionality for onboarding slide details UI.

### PR DESCRIPTION
**Pull request by CypherPoet**

### CODE REVIEW

**Comment by Christewart** [comment]
I can't see anything when i open up the onboarding pages

Screen Shot 2022-09-11 at 8 11 21 AM
Screen Shot 2022-09-11 at 8 11 21 AM

**Comment by CypherPoet** [comment]
@Christewart It looks like the text is there, but it's just super low contrast because you're in light mode. I should probably make a background that is better suited for dark text.

Also, I don't yet have routing hooked up for the complete button. I'm thinking it could go to the Wallet screen, but I'm open to other ideas as well.

**Comment by Christewart** [comment]
> Also, I don't yet have routing hooked up for the complete button. I'm thinking it could go to the Wallet screen, but I'm open to other ideas as well.

My understanding is the pages are ordered 

1. seed backup
2. wallet deposit
3. tor
4. sync

So shouldn't the complete button be on the last page (`sync`)?

**Comment by CypherPoet** [comment]
Yep, though I didn't add step 4 until after booking all this up: #200

File: wallet-server-ui/src/app/component/new-user-onboarding-card/new-user-onboarding-card.component.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  constructor() {}
+
+  @Input() card: NewUserOnboardingStep;
+
+  ngOnInit(): void {}
+
+  cardTitle(): string {
+    switch (this.card.onboardingStep) {
+      case 1:
+        return 'Backup Your Wallet Seed';
```
[the remark this answers is not in the document]
Comment (CypherPoet) [reply wallet-server-ui/src/app/component/new-user-onboarding-card/new-user-onboarding-card.component.ts:19]: @user411 👍  

https://github.com/bitcoin-s/bitcoin-s-ts/pull/200/files#diff-77d761ed736cbcdac16bb2b541b07b407eb219c8595ba616be3ecc4e2f430309R577

**Pull request merged**